news-aggregation/utils/commandUtility.ts

```diff
@@ -68,64 +101,80 @@ export class CommandUtility implements ICommandUtility {
 			// this.news
 			news
 		);
-		news = await techCrunchNewsSource.fetchNews(
-			this.read,
-			this.modify,
-			this.room,
-			this.http,
-			this.persistence
-		);
-
-		await techCrunchNewsSource.saveNews(this.persistence, this.persistenceRead);
-	}
 
-	public async getNewsFromSource() {
-		let news: NewsItem[] = [];
-
-		const techCrunchAdapter = new TechCrunchAdapter();
-		const techCrunchNewsSource = new NewsSource(
+		const newsStorage = new NewsItemPersistence(
 			this.app,
-			techCrunchAdapter,
-			// this.news
-			news
+			this.persistence,
+			this.persistenceRead
 		);
 
 		try {
-			news = await techCrunchNewsSource.getNews(
-				this.read,
-				this.modify,
-				this.room,
-				this.http,
-				this.persistence
-			);
+			// news = await techCrunchNewsSource.getNews(
+			// 	this.read,
+			// 	this.modify,
+			// 	this.room,
+			// 	this.http,
+			// 	this.persistence
+			// );
+			news = (await newsStorage.getAllNews()) as NewsItem[];
 			console.log('fetched!!', news, 'FETCHED FROM PERSISTENCE!');
+
+			let newsBlocks: Array<Array<Block>> = [];
+			for (const item of news) {
+				const newsBlock = await buildNewsBlock(item);
+				// newsBlocks.push(newsBlock);
+
+				await sendMessage(this.modify, this.room, this.sender, '', newsBlock);
```

Comment:
```suggestion
				await sendMessage(this.modify, this.room, await this.read.getUserReader().getAppUser() as IUser, '', newsBlock);
```
The sender of the news should be the app user (news-aggregation.bot) not the user who executed the command.

Reply:
Hey @umangutkarsh We need to implement this change as well.